### PR TITLE
PR for #1: Modify chips sold histogram to reflect percentages instead of counts

### LINKED ISSUE #1: Modify chips_sold.pdf histogram from displaying counts to percentages
For [Practice Task 1: Basic Functionality](https://github.com/gentzkow/lab-manual-archive/wiki/Practice-Task), modify code in the data folder so that the histogram in output/chips_sold.pdf displays percentages instead of counts.

### PR DESCRIPTION
**Pull request by linxicindyzeng**
Closes #1 

Changes made:

- In [clean_data.py](https://github.com/linxicindyzeng/template_archive/blob/cfbf06037ac506777f66df606f0551d984f3882b/data/code/clean_data.py), modify the code for chips_sold.pdf to reflect percentages instead of counts
- Run [data/make.py](https://github.com/linxicindyzeng/template_archive/blob/cfbf06037ac506777f66df606f0551d984f3882b/data/make.py) to update the data folder with the new histogram 
- Run [paper_slides/make.py](https://github.com/linxicindyzeng/template_archive/blob/cfbf06037ac506777f66df606f0551d984f3882b/paper_slides/make.py) to reflect the change in the paper

Deliverables:
- [New histogram for chips sold](https://github.com/linxicindyzeng/template_archive/blob/48e472a95ab5336cdac24edc411e113d29aabb7a/data/output/chips_sold.pdf)
- [Paper updated with new histogram for chips sold](https://github.com/linxicindyzeng/template_archive/blob/48e472a95ab5336cdac24edc411e113d29aabb7a/paper_slides/output/paper.pdf)

Draftable Difference:
- See [here](https://draftable.com/compare/apMIPhlQTimC)

@ShiqiYang2022 Hi Shiqi, please let me know if anything needs to be modified. Thanks for reviewing!

### CODE REVIEW

File: paper_slides/code/slides.fdb_latexmk
Comment (ShiqiYang2022) [inline paper_slides/code/slides.fdb_latexmk:1]: @linxicindyzeng Why this file should be committed? I think this might be a commit by mistake.

File: paper_slides/code/slides.pdf
Comment (ShiqiYang2022) [inline paper_slides/code/slides.pdf:1]: @linxicindyzeng Why this file should be committed? I think this might be a commit by mistake.

File: paper_slides/code/slides.synctex.gz
Comment (ShiqiYang2022) [inline paper_slides/code/slides.synctex.gz:1]: @linxicindyzeng Why this file should be committed? I think this might be a commit by mistake.

File: paper_slides/code/slides.log
Comment (ShiqiYang2022) [inline paper_slides/code/slides.log:1]: @linxicindyzeng Why this file should be committed? I think this might be a commit by mistake.

File: setup/download_stata_ado.do
Comment (ShiqiYang2022) [inline setup/download_stata_ado.do:1]: @linxicindyzeng Why this file should be committed? I think this might be a commit by mistake.

File: texput.log
Comment (ShiqiYang2022) [inline texput.log:1]: @linxicindyzeng Why this file should be committed? I think this might be a commit by mistake.

**Review by ShiqiYang2022 — CHANGES_REQUESTED** [review]
@linxicindyzeng Thanks for all of the work here! 

I have several comments, some of them might not be documented in the RA manual, but they are our lab's routines in doing a pull request.

1. Please rename your Pull request to "Pull request for #1: Modify chips sold histogram to reflect percentages instead of counts" or "PR for #1: ...". Every PR's title should have such format.
2. Every PR should begin with "Closes #IssueNumber." so that it will links to the related issue. See [here](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue). Please revise your https://github.com/linxicindyzeng/template_archive/pull/2#issue-2496200276 accordingly.
3. Please revise your https://github.com/linxicindyzeng/template_archive/pull/2#issue-2496200276 to display files' permanent links (ref [here](https://docs.github.com/en/repositories/working-with-files/using-files/getting-permanent-links-to-files)) on the branch (instead of `master`).
4. The link for the histogram and the paper is expired in https://github.com/linxicindyzeng/template_archive/pull/2#issue-2496200276, please double check.
5. When the issue request changes in the input of the paper, it would be great to deliver a [draftable difference](https://www.draftable.com/compare). Could you please revise your https://github.com/linxicindyzeng/template_archive/pull/2#issue-2496200276 accordingly to include a draftable difference link?
6. I left some other comments about the files that should not be committed in the thread.

After you've addressed those issues, please tag me again and I'll have a final review. Thanks!

**Comment by linxicindyzeng** [comment]
@ShiqiYang2022  Hi Shiqi, thank you so much for the careful review and comments! I've made modifications as suggested. Please let me know if more changes are needed.

File: paper_slides/code/slides.fdb_latexmk
Comment (ShiqiYang2022) [reply paper_slides/code/slides.fdb_latexmk:1]: @linxicindyzeng  Usually, the conversation thread should be resolved by the reviewer. What we do is,

If the assignee accept the review, he addressed in a commit and tag the reviewer with providing the exact commit hash number. For example, see [here](https://github.com/gslab-econ/newsworthy/pull/249#discussion_r1708144165).

If the assignee disagree with the review, he replies the reviewer with the specific reason. For example, see [here](https://github.com/gentzkow/GentzkowLabTemplate/pull/14#discussion_r1706077612).

The reviewer then decides whether the conversation is good to go or not. We don't recommend the assignee to resolve the conversation himself.

**Review by ShiqiYang2022 — APPROVED** [review]
Thanks @linxicindyzeng! The changes all looks good from my side hence I'm approving.

I have one comment in a re-opened thread, it's also about our work routine in a PR.

File: paper_slides/code/slides.fdb_latexmk
Comment (linxicindyzeng) [reply paper_slides/code/slides.fdb_latexmk:1]: @ShiqiYang2022 Sorry! You're right, that makes much more sense. Thank you for explaining it in detail.

**Comment by ShiqiYang2022** [comment]
I think now you can feel free to "squash and merge" this branch back to `master`. After we merged back to `master`, please post a summary comment in #1 to summarize what you've done in the issue and its associated PR.

You can feel free to refer the summary comment for my 1st issue in our lab as an example, see [here](https://github.com/gslab-econ/ad-price-drivers/issues/706#issuecomment-1640744067).

File: paper_slides/code/slides.fdb_latexmk
Comment (ShiqiYang2022) [reply paper_slides/code/slides.fdb_latexmk:1]: @linxicindyzeng No worries, that's exactly what a practice task meant for! I am now resolving the thread.

**Pull request merged**
